packages/ethereum-contracts/contracts/agreements/ConstantFlowAgreementV1.sol

```diff
@@ -457,7 +463,8 @@ contract ConstantFlowAgreementV1 is
         // still be able to recreate the hook behavior. This logic should exist in the NFT contract though.
```

Comment:
can we remove the mentioning of NFT please.

Reply:
Let's play the game of not assuming what the hooked contract is about in this context.